yogstation/code/modules/mob/living/simple_animal/friendly/goats.dm

```diff
@@ -0,0 +1,347 @@
+/mob/living/simple_animal/hostile/retaliate/goat/clown
+	name = "Gary the Goat"
+	desc = "Rather kick your butt then tell jokes."
+	icon = 'yogstation/icons/mob/clownpets.dmi'
+	icon_state = "clown_goat"
```

Comment:
All of the goats have their own DMIs, I recommend keeping icon_state, icon_living, and icon_dead the same across all of them.